modules/friendlyshared.js

```diff
@@ -75,49 +77,53 @@ Twinkle.shared.callback = function friendlysharedCallback() {
 	Window.display();
 };
 
-Twinkle.shared.standardList = [
-	{
-		label: '{{Shared IP}}: standard shared IP address template',
-		value: 'Shared IP',
-		tooltip: 'IP user talk page template that shows helpful information to IP users and those wishing to warn, block or ban them'
-	},
-	{
-		label: '{{Shared IP edu}}: shared IP address template modified for educational institutions',
-		value: 'Shared IP edu'
-	},
-	{
-		label: '{{Shared IP corp}}: shared IP address template modified for businesses',
-		value: 'Shared IP corp'
-	},
-	{
-		label: '{{Shared IP public}}: shared IP address template modified for public terminals',
-		value: 'Shared IP public'
-	},
-	{
-		label: '{{Shared IP gov}}: shared IP address template modified for government agencies or facilities',
-		value: 'Shared IP gov'
-	},
-	{
-		label: '{{Dynamic IP}}: shared IP address template modified for organizations with dynamic addressing',
-		value: 'Dynamic IP'
-	},
-	{
-		label: '{{Static IP}}: shared IP address template modified for static IP addresses',
-		value: 'Static IP'
-	},
-	{
-		label: '{{ISP}}: shared IP address template modified for ISP organizations (specifically proxies)',
-		value: 'ISP'
-	},
-	{
-		label: '{{Mobile IP}}: shared IP address template modified for mobile phone companies and their customers',
-		value: 'Mobile IP'
-	},
-	{
-		label: '{{Whois}}: template for IP addresses in need of monitoring, but unknown whether static, dynamic or shared',
-		value: 'Whois'
+Twinkle.shared.initialize = function() {
+	if( typeof Twinkle.shared.standardList === 'undefined' ) {
+		Twinkle.shared.standardList = [
```

Comment:
Something like: the data contains an entry which exactly matches the format of this array, e.g. 
```
[
    {
        label: '{{ergojduhd}}: lorem ipsum dolor sit amet',
        value: 'ergojduhd',
        tooltip: 'consectetur adipiscing elit'
    },
    ...
]
```
Then these are accompanied by strings like
`'tw-sharedip-edu-template-name': 'Shared IP edu'`
which would be used below in place of the hard-coded template name.

Obviously this is deficient. Most wikis will not need this semi-hard-coded opt-out. In the end, the enable/disable callback and form validation code would be able to be localised as well. Whether or not you want to do that at this point is up to you, I guess.